### DISCUSSION TITLE
How to install a package with options?

### DISCUSSION THREAD

**Discussion by mostafa-rz** [mlocati/docker-php-extension-installer#735] [Q&A] [answered]
Hi,
I want to install gd with some options, for example `gd --with-freetype --with-jpeg --with-webp`. How can i pass these options to the `install-php-extensions gd`?

Thanks.

**Comment by mlocati** [comment, marked as answer]
When I add support for a PHP extension, I do my best to enable every possible feature.

For GD, for example, we [already set `--with-freetype --with-jpeg --with-webp`](https://github.com/mlocati/docker-php-extension-installer/blob/2.1.19/install-php-extensions#L2264-L2308).
